src/context/LogContext/index.tsx

```diff
@@ -107,6 +108,9 @@ const LogContextProvider: React.FC<LogContextProviderProps> = ({
   const [zebraStriping, setZebraStriping] = useState(
     Cookie.get(ZEBRA_STRIPING) === "true",
   );
+  const [wordWrapFormat, setWordWrapFormat] = useState(
```

Comment:
It could be cool to have a custom hook to manage all of our cookie states. New ticket maybe?